### PR TITLE
- added video listing pages and player

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**
- videos listing page
 image
- videos player page
 image
- made it compatible to mobile
 image

### CODE REVIEW

File: src/shared/card/VideoCard.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+const VideoCard = () => {
+  const { videos, loader } = useVideo();
+  return (
+    <div className="video-listing-card bg-theme-changer">
+      {loader && <div className="flex-center"> Loading the Videos </div>}
+      {videos.map((item) => {
+        return (
+          <div key={item._id}>
+            <Link to={`/player/${item._id}`}>
+              <img src={item.img} className="video-images" alt={item.title} />
```
Comment (paarthbajaj) [inline src/shared/card/VideoCard.js:13]: use object-fit and object-position properties, your thumbnails will look much better

File: src/shared/sidebar/sidebar.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+.sidebar-icon {
+    font-size: 1.4rem;
+}
+
+.sidebar-text {
+    margin-left: 1rem;
+    font-size: 1.1rem;
+    font-weight: 600;
```
Comment (paarthbajaj) [inline src/shared/sidebar/sidebar.css:26]: use the class you have created in index.css

**Review by paarthbajaj — APPROVED** [review]
The code looks good. I have added some comments. Please have a look at them. Also, I think the navbar could be added to the components folder. If it is not possible, then leave it, no worries. Keep it up!

File: src/context/videosContext.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  const [videos, setVideos] = useState([]);
+  const [loader, setLoader] = useState(true);
+
+  const fetchVideos = async () => {
+    try {
+      const { data } = await axios.get("/api/videos");
+      setLoader(false);
+      setVideos(data.videos);
+    } catch (error) {
+      console.log(error);
```
Comment (hussiantabha) [inline src/context/videosContext.js:16]: Remove console.log

**Review by hussiantabha — APPROVED** [review]
Everything works fine

File: src/shared/card/VideoCard.js (hunk 1 above)
Comment (ajit-sharmaGH) [reply src/shared/card/VideoCard.js:13]: Okk

File: src/shared/sidebar/sidebar.css (hunk 2 above)
Comment (ajit-sharmaGH) [reply src/shared/sidebar/sidebar.css:26]: Yes yes, I missed

File: src/context/videosContext.js (hunk 3 above)
Comment (ajit-sharmaGH) [reply src/context/videosContext.js:16]: Okk

**Pull request merged**